packages/dnslink-api/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16.3.0-alpine
+
+WORKDIR /usr/app
+
+COPY package.json .
+RUN yarn --no-lockfile
```

Comment:
This is interesting. Why are we running with `--no-lockfile`? I usually hear arguments for always using the lockfile without updating it (in prod, you obviously update it during dev), as that guarantees that you'll be running the same module versions in production that you tested with during your CI run. Running without a lockfile will always get you the latest compatible versions but has a much weaker guarantee that those version will work sufficiently well together.

Reply:
yeah that's due to how our package system works, we have one lock file for whole skynet-webportal repo becuase we're using workspaces but I guess we should really just treat it as a monorepo with multiple packages and have separate lock for each package - we can put that on the roadmap for sometime in the future (making new issue for that now)